examples/tts/conf/vits_44100.yaml

```diff
@@ -202,12 +200,18 @@ trainer:
   check_val_every_n_epoch: 1
 
 exp_manager:
-  exp_dir: ???
+  exp_dir: ../exps/vits_hifitts
```

Comment:
We set this to `null` by default for other models, can you change this to match as well?